src/App.js

```diff
@@ -30,20 +30,14 @@ class App extends React.Component {
   updateResults = (request) => {
 
     let historyChecker = ls.get('history') || [];
-    console.log('in', historyChecker);
     let historyUpdate;
 
-    console.log(historyChecker.indexOf(request));
-
     if(historyChecker.indexOf(request) === -1) {
       historyChecker.unshift(request);
-      console.log('added');
     }
     
     historyUpdate = historyChecker;
 
```

Comment:
- When I try and search for data I get an error about your App.js line 87 having an error with this.setState
- Would like to see more added to your README